extendr-api/src/scalar/rcplx.rs

```diff
@@ -0,0 +1,186 @@
+use crate::scalar::macros::*;
+use crate::scalar::Rfloat;
+use crate::*;
+use std::convert::TryFrom;
+use std::ops::{Add, Div, Mul, Neg, Sub};
+use std::ops::{AddAssign, DivAssign, MulAssign, SubAssign};
+
+// #[cfg(feature="num-complex")]
+pub type C64 = num_complex::Complex<f64>;
```

Comment:
I know it is against the naming convention, but shouldn't it be `c64` inline with `f64` and `i64`/`u64`?

Reply:
Rust likes types to start with a capital letter. But we can override this with a #[allow()]

Reply:
I know, I am just curious and asking for opininons. I am ok with either options.